extensions/wikia/SpecialPageViews/js/SpecialPageViews.js

```diff
@@ -0,0 +1,473 @@
+var SponsorshipDashboard = function(){
```

Comment:
This file will require some linting and maybe we can make AMD module from this code